src/liseq.py

```diff
@@ -31,4 +31,18 @@ def liseq(x: Sequence[Any]) -> list[int]:
     """
     best: list[int] = []
     # Explore all alternatives
+    for i in range(len(x)):
+        for j in range(i, len(x)):
+            current_list=[i]
+            if is_increasing([x[i],x[j]]):
```

Comment:
If you are just comparing two numbers, `is_increasing` is overkill. Using `x[i] <= x[j]` would do the same thing.